Open Judge System/Data/OJS.Data/Migrations/DefaultMigrationConfiguration.cs

```diff
@@ -357,7 +357,7 @@ protected void SeedSubmissionTypes(OjsDbContext context)
                 {
                     Name = "C++ Zip File",
                     CompilerType = CompilerType.CPlusPlusZip,
-                    AdditionalCompilerArguments = "-pipe -mtune=generic -O3 -static-libgcc -static-libstdc++",
```

Comment:
We decided on it because of a specific problem used in the C++ course where users have to invent a way to hardcode a large array in the program in order to bypass reading times (it's an outside the box kinda problem admittedly), but if they do it in a suboptimal way it takes the compiler approximately 10 seconds to optimize the code, while the actual runtime performance improvement is negligible (~1 milisecond or less) . Given that any submit will not be run more than once per compilation and the abysmal gains in performance, it makes no sense to spend the extra optimization time. We're dropping the optimization only for the C++ Zip strategy where large files can be expected, the standard C++ strategy will remain the same.